patches/minecraft/net/minecraft/potion/Potion.java.patch

```diff
@@ -88,6 +88,18 @@
 +        ret.add(new net.minecraft.item.ItemStack(net.minecraft.init.Items.field_151117_aB));
 +        return ret;
 +    }
++    
++    /**
++     * Return an index that will override the potion's default sort order. This is useful for potions with colors 
```

Comment:
This isn't very clear if you don't look at the usage. Maybe add "The comparison between indices takes precedence over the vanilla logic. If two effects compare equal by index, vanilla takes over." after the first sentence.